src/promptflow/promptflow/_cli/_pf_azure/_run.py

```diff
@@ -133,17 +131,20 @@ def add_parser_run_stream(subparsers):
 # Stream run logs:
 pfazure run stream --name <name>
 """
-    run_stream_parser = subparsers.add_parser(
-        "stream",
+    add_params = [
+        add_param_run_name,
+        _set_workspace_argument_for_subparsers,
+    ] + base_params
+
+    activate_action(
+        name="stream",
         description="A CLI tool to stream run logs to the console.",
         epilog=epilog,
-        help="Stream run logs to the console.",
+        add_params=add_params,
+        subparsers=subparsers,
+        help_message="Show a run.",
```

Comment:
why change the help message for stream?

Reply:
Good catch, was refactoring the stream parser functio. Recovered the help message.